Quotient/room.cpp

```diff
@@ -3444,3 +3447,36 @@ void Room::startVerification()
     d->pendingKeyVerificationSession = new KeyVerificationSession(this);
     emit d->connection->newKeyVerificationSession(d->pendingKeyVerificationSession);
 }
+
+QJsonArray Room::exportMegolmSessions()
+{
+    QJsonArray sessions;
+    for (auto &[key, value] : d->groupSessions) {
```

Comment:
```suggestion
    for (const auto& [key, value] : d->groupSessions) {
```
just to make sure :)